Breezy-Traveler/UserModel.swift

```diff
@@ -18,6 +18,8 @@ struct BTUser: Codable {
     static func getStoredUser() -> BTUser {
         let userPersistence = UserPersistence()
         guard let currentUser = userPersistence.getCurrentUser() else {
+            
+            // FIXME: crashing the app
```

Comment:
I wouldn't think this needs a fix me. this `getStoredUser()` should never be called without checking if a user is logged in.